Sources/DatadogObjc/OpenTracing/OTTracer+objc.swift

```diff
@@ -5,11 +5,12 @@
  */
 
 import Foundation
-public let OTFormatHTTPHeaders = "OTFormatHTTPHeaders"
+// TODO: RUMM-917 this can't be exposed to ObjC but it is required in `inject(format:...)` method
+// Currently `inject` method is not usable from ObjC
+public let OTFormatHTTPHeaders = "OTFormatHTTPHeaders" // swiftlint:disable:this non_exposed_objc
```

Comment:
Constants can be exposed in the way used in `ExampleAppDelegate`: https://github.com/DataDog/dd-sdk-ios/blob/53109cfe918bc2d17ac75e95f681ce7fc82ec60b/Datadog/Example/ExampleAppDelegate.swift#L94-L98

I couldn't find a better way.

Reply:
that's what i thought too but i'm not sure of naming, what do you prefer as names?

Reply:
This particular constant is named by its counterpart in [opentracing-objc](https://github.com/opentracing/opentracing-objc/blob/9d73066311e511e6649bd24c17a5e1689eb88184/Pod/Classes/OTTracer.h#L8-L17):

```objc
/**
 * The TextMap format identifier for OTTracer#inject:format:carrier: and
 * OTTracer#extractWithFormat:carrier:
 *
 * The carrier for OTFormatTextMap must be an NSMutableDictionary* instance for
 * `inject` and an NSDictionary* instance for `extract`.
 *
 * @see http://opentracing.io/propagation/
 */
FOUNDATION_EXPORT NSString *const OTFormatTextMap;
```

Ideally, we should expose it as an Obj-c constant, but due to this Swift <> Objc bridging limitation maybe we can create an artificial namespace of `OT` → `OT.FormatTextMap`?